cmd/prReview.go

```diff
@@ -9,14 +9,17 @@ import (
 	"github.com/gsoares85/code-guardian/internal/github"
 	"github.com/gsoares85/code-guardian/internal/openai"
 	"github.com/spf13/cobra"
+	"os"
+	"path/filepath"
 	"strconv"
 )
 
 // prReviewCmd represents the prReview command
 var prReviewCmd = &cobra.Command{
-	Use:   "pr-review [owner] [repo] [pr-number]",
+	Use:   "pr-review [owner] [repo] [pr-number] [flags]",
 	Short: "Analysis of a pull request",
 	Long:  `Use this command to perform a complete code review of a pull request.`,
+	Args:  cobra.ExactArgs(3),
 	Run: func(cmd *cobra.Command, args []string) {
```

Comment:
The complexity of the `Run` function has increased significantly. It's a good practice to separate the concerns. For example, you can put the generation of output content into a different function.